src/CtrlS/RoundState.java

```diff
@@ -67,20 +82,31 @@ private int calculateCoin() {
         } else if (timeDifferenceInSeconds <= 100) {
             timeBonus = 10;
         }
-        coin += timeBonus;
+
 
         if (levelBonus2 > 1){
             coin = (int) (coin * levelBonus2);
             Core.getLogger().info("item level bonus occurs (" + (int) ((levelBonus2 - 1) * 100) + "%).");
         }
 
+        //코인 증가
+        if(roundHealth > 0){
```

Comment:
roundHealth 변수와 getter를 만들지 않고, 단순화 하는 게 어떨까요?

```suggestion
        if(currState.getLivesRemaining() > 0){
```